javascript/organizing_your_javascript_code/factory_functions_and_module_pattern.md

```diff
@@ -305,3 +305,4 @@ This section contains questions for you to check your understanding of this less
 This section contains helpful links to other content. It isn't required, so consider it supplemental.
 
 - [Learning JavaScript Design Patterns by Addy Osmani and Lydia Hallie](https://www.patterns.dev/)
+- [This video](https://www.youtube.com/watch?v=ndadGZuFTvI) provides a helpful example of the module pattern in action.
```

Comment:
```suggestion
- [This video from DevTuts about the revealing module pattern](https://www.youtube.com/watch?v=ndadGZuFTvI) provides a helpful example of the module pattern in action.
```

I think the link should be more descriptive. Otherwise it looks good and it's a nice resource.

Reply:
I'm sorry but I just noticed that I  missed it the first time I went through this lesson. I actually just realized there already is a video on the modular pattern. What do you think this one adds that isn't covered in that one?

Reply:
Hey @ManonLef, this video just seemed slightly more helpful to me when going through and learning about the module pattern. Therefore, I believed it would be useful as an additional resource for another example of the module pattern.

Reply:
I do see how it could be nice as a small video on how the IIFE works but I'm not sure how useful that would be at the end of this lesson when it's been covered by various resources in depth already. I will ask the other @TheOdinProject/javascript maintainers for their opinion.

Reply:
Ah yes, as an introduction possibly could work better.

Reply:
I'd agree that it may not be necessary if it's essentially covering topics already covered in the lesson/assignments, just in a slightly different way. I do think there could be merit to having an additional resource re-cover topics in a different way, such as an interactive tutorial/game similar to Flexbox Froggy and such.

While it's great that it helped solidify concepts, I think something like this is more on the user to search out/stumble upon should they wish.

Reply:
Hm, yeah, that makes more sense. Should the PR be closed then?

Reply:
I will close it. Thank you nonetheless!